### PR TITLE
[OMA-850] Prepare standalone module for HyBid extension

### PR DESCRIPTION
**Pull request by eros902002**

### CODE REVIEW

File: appmonet.hybid/src/main/java/com/monet/bidder/AppMonet.java [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+     * @param context               The context ({@link Context}) of your application.
+     * @param appMonetConfiguration The application configurations needed to initialize the sdk.
+     */
+    public static void init(Context context, AppMonetConfiguration appMonetConfiguration) {
+        AppMonetConfiguration internalConfiguration = appMonetConfiguration;
+        if (appMonetConfiguration == null) {
+            internalConfiguration = new AppMonetConfiguration.Builder().build();
+        }
+        if (TextUtils.isEmpty(internalConfiguration.applicationId)) {
+
```
Comment (faresbh) [inline appmonet.hybid/src/main/java/com/monet/bidder/AppMonet.java:94]: You don't need to throw an error here instead of empty block ?

**Review by faresbh — APPROVED** [review]

File: appmonet.hybid/src/main/java/com/monet/bidder/AppMonet.java (hunk 1 above)
Comment (eros902002) [reply appmonet.hybid/src/main/java/com/monet/bidder/AppMonet.java:94]: Yes @faresbh you're right. Let me fix it.

File: appmonet.hybid/src/main/java/com/monet/bidder/AppMonet.java [hunk 2]
```diff
@@ -91,7 +91,7 @@ public static void init(Context context, AppMonetConfiguration appMonetConfigura
             internalConfiguration = new AppMonetConfiguration.Builder().build();
         }
         if (TextUtils.isEmpty(internalConfiguration.applicationId)) {
-
+            Logger.e(TAG, "Error initialising the AppMonet SDK. Invalid application ID provided");
         } else {
```
Comment (faresbh) [inline appmonet.hybid/src/main/java/com/monet/bidder/AppMonet.java:95]: Looks good

**Review by faresbh — APPROVED** [review]

File: appmonet.hybid/src/main/java/com/monet/bidder/AppMonet.java (hunk 1 above)
Comment (eros902002) [reply appmonet.hybid/src/main/java/com/monet/bidder/AppMonet.java]: Fixed : )

**Pull request merged**